puppet/modules/kubernetes/templates/kubelet.service.erb

```diff
@@ -83,7 +83,7 @@ ExecStart=<%= scope['kubernetes::_dest_dir'] %>/kubelet \
     @cgroup_kube_reserved << "memory=#{@cgroup_kube_reserved_memory}" unless @cgroup_kube_reserved_memory.nil? or @cgroup_kube_reserved_memory == 'nil'
     if @cgroup_kube_reserved.length > 0
 -%>
-  "--kube-reserved=<%= @cgroup_kube_reserved.join(',') %>" \
```

Comment:
Why remove quoting?

Reply:
because none of the other flags were quoted and it looked a bit odd when viewing the systemd unit - it's only cosmetic